index.js

```diff
@@ -32,9 +28,18 @@ app.get('/todos', (request, response) => {
   });
 });
 
+app.get('/todos/:id',(request, response)=> {
+  connection.query('SELECT * FROM todos WHERE todoId = ?', [request.params.id], (error, data) => {
+    if (data[0] === undefined) {
+      return response.status(404).json({status: 404, data: error});
```

Comment:
Hi, I think it's a good idea to use 'no data' handler, but in case of no data I don't think it's gonna be an error it should give 'no such entry' message and you should add error handler in addition to this that will return if there was an error in the query itself.